src/main/resources/reference.conf

```diff
@@ -1384,6 +1381,14 @@ literature {
         format = ${common.output-format}
         path = ${literature.output}"/literatureSentences"
       }
+      failed-cooccurrences {
+        format = ${common.output-format}
+        path = ${literature.output}"/cooccurrences"
```

Comment:
@remo87 , please, note that _failed coocurrences_ should output to 
```
${literature.output}"/failedCooccurrences"
```

Reply:
true! it has been fixed in the latest push